src/components/index.ts

```diff
@@ -9,7 +9,7 @@ export {
   AccordionIconVariant,
   AccordionIcon,
 } from './Accordion';
-export { Button, ButtonVariant } from './Button';
+export { Button, ButtonVariant, ButtonColor, ButtonSize } from './Button';
 export { Page, PageHeader, PageContent, PageColor } from './Page';
```

Comment:
Could SvgIcon be used as a standalone component as well? If so, you should export it here :)

Reply:
SvgIcon could indeed be exposed as a standalone component. I had a discussion with @mjulstein and @TomasEng about this and we seemed to agree that the IconComponent should not be exposed for now in order to more tightly enforce the design narrative of the design-system. Exposing the icon-component also removes some incentive of creating new components in the design-system.